### PR TITLE
Feature/css

### PR DESCRIPTION
**Pull request by lee-benjamin**

### CODE REVIEW

File: src/js/twitter.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  tweetDiv.appendChild(a);
+}
+
+
+function appendTweet(tweet, color) {
+  // Each tweet will live in a <div>
+  var tweetDiv = document.createElement("div"); // The root element of the tweet
+  tweetDiv.setAttribute("class", "tweetDiv");
+  if (color) {
+    tweetDiv.setAttribute("id", "lightTweet");
```
Comment (kev-chan) [inline src/js/twitter.js]: ids should be unique

File: src/styles/style.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+
+#getTimelineButton {
+    margin: 25px;
+}
+
+.tweetDiv {
+    border-bottom: 1px black solid;
+}
+
+#lightTweet {
```
Comment (kev-chan) [inline src/styles/style.css]: can you write this in css?

File: src/styles/style.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
+#divTimeline {
```
Comment (kev-chan) [inline src/styles/style.css]: could practice is to use classes for rules more than ids

File: src/styles/style.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+    background-color: #e9e9e9;
+}
+
+.img-circle {
+    border-radius: 50%;
+    height: 60px;
+    width: 60px;
+}
+
+figure {
```
Comment (kev-chan) [inline src/styles/style.css]: use a class instead of the element name

File: src/js/twitter.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+  var element = document.getElementById("divTimeline");
+  element.appendChild(tweetDiv);
+}
+
+function getHomeTimeline() {
+  var xhttp = new XMLHttpRequest();
+  xhttp.onreadystatechange = function() {
+   document.getElementById("divTimeline").innerHTML = "";
+    var stylesheet = document.styleSheets[0];
+    stylesheet.insertRule("#divTimeline {border-style: solid; border-width: 2px;}", 0);
```
Comment (kev-chan) [inline src/js/twitter.js]: you should not need to insert a rule dynamically

File: src/js/main.js [hunk 6]
```diff
@@ -2,9 +2,45 @@ var http = require("http");
 var fs = require("fs");
 
 http.createServer(function (request, response) {
-    fs.readFile("src/index.html", function(err, data) {
-        response.writeHead(200, {'Content-Type': 'text/html'});
-        response.write(data);
+    console.log(request.url);
```
Comment (kev-chan) [inline src/js/main.js]: is this console log still needed?

File: src/js/twitter.js (hunk 1 above)
Comment (lee-benjamin) [reply src/js/twitter.js]: fixed

File: src/styles/style.css (hunk 3 above)
Comment (lee-benjamin) [reply src/styles/style.css]: fixed

File: src/styles/style.css (hunk 4 above)
Comment (lee-benjamin) [reply src/styles/style.css]: fixed

File: src/js/main.js (hunk 6 above)
Comment (lee-benjamin) [reply src/js/main.js]: done

File: src/js/twitter.js [hunk 7]
```diff
@@ -0,0 +1,101 @@
...
+  bodyDiv.innerHTML = tweet.text;
+  a.appendChild(bodyDiv);
+  tweetDiv.appendChild(a);
+}
+
+
+function appendTweet(tweet, color) {
+  // Each tweet will live in a <div>
+  var tweetDiv = document.createElement("div"); // The root element of the tweet
+  if (color) {
```
Comment (kev-chan) [inline src/js/twitter.js]: can you move the coloring strictly inside css and not based on a conditional class?

File: src/js/twitter.js [hunk 8]
```diff
@@ -0,0 +1,101 @@
...
+    tweetDiv.setAttribute("class", "tweetDiv lightTweet");
```
Comment (kev-chan) [inline src/js/twitter.js]: you can do this for multiple classes or get the classlist in javascript and add to it

File: src/js/twitter.js [hunk 9]
```diff
@@ -0,0 +1,101 @@
...
+
+  addImage(tweetDiv, tweet);
+  // create a div to hold the date and text
+  var tweetContent = document.createElement("div");
+  tweetContent.setAttribute("class", "tweetContent");
+  addTime(tweetContent, tweet);
+  addText(tweetContent, tweet);
+  tweetDiv.appendChild(tweetContent);
+  // clear block div for styling
+  var clearfix = document.createElement("div");
```
Comment (kev-chan) [inline src/js/twitter.js]: you can put the clearfix on an existing element as opposed to adding an explicit div only for clearfix

File: src/js/twitter.js [hunk 10]
```diff
@@ -0,0 +1,101 @@
...
+  xhttp.onreadystatechange = function() {
+    var divTimeline = document.getElementById("divTimeline");
+    divTimeline.innerHTML = ""; // Clearing previous text
+    if (this.readyState == XMLHttpRequest.DONE)  {
+      if (this.status == 200) {
+        var obj = JSON.parse(this.responseText);
+        for (var i=0;i<obj.length;i++) {
+            appendTweet(obj[i], i % 2 == 0);
+        }
+        divTimeline.setAttribute("style", "border-style: solid; border-width: 2px;");
```
Comment (kev-chan) [inline src/js/twitter.js]: same comment as before, let's move this all in css versus having an inline style

Comment (kev-chan) [reply src/js/twitter.js]: 1px

File: src/js/twitter.js (hunk 8 above)
Comment (lee-benjamin) [reply src/js/twitter.js]: fixed

File: src/js/twitter.js (hunk 10 above)
Comment (lee-benjamin) [reply src/js/twitter.js]: done

File: src/js/twitter.js (hunk 9 above)
Comment (lee-benjamin) [reply src/js/twitter.js]: apparently clearfix + float is gauche so I switched to display: inline-block

File: src/js/twitter.js (hunk 7 above)
Comment (lee-benjamin) [reply src/js/twitter.js]: done

**Pull request merged**